src/main.ts

```diff
@@ -36,46 +86,22 @@ export default function vitePluginArraybuffer(): PluginOption {
         ])`;
       }
       if (id.endsWith("?arraybuffer&base64")) {
-        const file = id.slice(0, -12)
+        const file = id.slice(0, -19)
```

Comment:
Ah, this was the mistake I missed!

Reply:
When I re-maintained this plugin it took me a while to understand what this was for 🫡